packages/manager/src/features/Linodes/LinodesDetail/LinodeConfigs/LinodeConfigDialog.tsx

```diff
@@ -165,23 +168,51 @@ const interfacesToState = (interfaces?: Interface[]) => {
     return defaultInterfaceList;
   }
   const interfacesPayload = interfaces.map(
-    ({ ipam_address, label, purpose }) => ({ ipam_address, label, purpose })
+    ({
+      id,
+      ipam_address,
+      ipv4,
+      label,
+      primary,
+      purpose,
+      subnet_id,
+      vpc_id,
+    }) => ({
+      id,
+      ipam_address,
+      ipv4,
+      label,
+      primary,
+      purpose,
+      subnet_id,
+      vpc_id,
+    })
```

Comment:
nitpick: I see what we're trying to do here -- maybe worth excluding the two properties and then just using rest? (I think these are the properties we want)
```suggestion
    ({
      ipv6, ip_ranges, ...rest
    }) => rest
```

Reply:
Going to keep this as-is for now since the linter will give an unused variable warning for the two unused ones currently, but once VPC is in GA and `ipv6` and `ip_ranges` are supported down the line we should be able to remove this bit entirely (just padding `interfaces` without doing any mapping).